platform/ios/src/MGLMapboxEvents.m

```diff
@@ -131,35 +148,26 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 - (void)updateNonDisablingConfigurationValues {
     if ([[[[NSUserDefaults standardUserDefaults] dictionaryRepresentation] allKeys] containsObject:MGLTelemetryAccessTokenKey]) {
         NSString *telemetryAccessToken = [[NSUserDefaults standardUserDefaults] objectForKey:MGLTelemetryAccessTokenKey];
-        NSUserDefaults.mme_configuration.mme_accessToken = telemetryAccessToken;
+        self.eventsManager.configuration.accessToken = telemetryAccessToken;
     }
 }
 
 - (void)updateDisablingConfigurationValues {
     BOOL collectionEnabled = [NSUserDefaults.standardUserDefaults boolForKey:MGLMapboxMetricsEnabledKey];
-    NSUserDefaults.mme_configuration.mme_isCollectionEnabled = collectionEnabled;
-    
+    self.eventsManager.configuration.isCollectionEnabled = collectionEnabled;
     [self.eventsManager pauseOrResumeMetricsCollectionIfRequired];
 }
 
 + (void)setupWithAccessToken:(NSString *)accessToken {
 
     MGLMapboxEvents *events = [MGLMapboxEvents sharedInstance];
-
-    // From https://github.com/mapbox/mapbox-events-ios
-    NSString *semanticVersion = [NSBundle mgl_frameworkInfoDictionary][@"MGLSemanticVersionString"];
-    NSString *shortVersion = [NSBundle mgl_frameworkInfoDictionary][@"CFBundleShortVersionString"];
-    NSString *sdkVersion = semanticVersion ?: shortVersion;
-
     MMEEventsManager *eventsManager = MMEEventsManager.sharedManager;
 
     // It is possible that an alternative access token was already set on this instance when the class was loaded
     // Use it if it exists
     NSString *resolvedAccessToken = [MGLMapboxEvents sharedInstance].accessToken ?: accessToken;
 
-    [eventsManager initializeWithAccessToken:resolvedAccessToken
-                               userAgentBase:MGLAPIClientUserAgentBase
```

Comment:
@alfwatt Do you know If these userAgent/host are still necessary (relative to reformed user agent work)?